src/components/PersonRow/PersonRow.tsx

```diff
@@ -0,0 +1,55 @@
+import { useCallback, useMemo } from 'react';
+import { PersonLink } from './PersonLink';
+import classNames from 'classnames';
+import { Person } from '../../types';
+
+type Props = {
+  personToRender: Person;
+  slug: string | undefined;
+  people: Person[];
+};
+
+export const PeopleRow: React.FC<Props> = ({
+  personToRender,
+  slug,
+  people,
+}) => {
+  const findParentInTable = useCallback(
+    (parentName: string | null) => {
+      return people.find(person => person.name === parentName);
+    },
+    [people],
+  );
+  const mother = useMemo(() => {
+    return findParentInTable(personToRender.motherName);
+  }, [findParentInTable, personToRender.motherName]);
+  const father = useMemo(() => {
+    return findParentInTable(personToRender.fatherName);
+  }, [findParentInTable, personToRender.fatherName]);
```

Comment:
u need more blank lines. it will be more readable
```suggestion
  const findParentInTable = useCallback(
    (parentName: string | null) => {
      return people.find(person => person.name === parentName);
    },
    [people],
  );
  
  const mother = useMemo(() => {
    return findParentInTable(personToRender.motherName);
  }, [findParentInTable, personToRender.motherName]);
  
  const father = useMemo(() => {
    return findParentInTable(personToRender.fatherName);
  }, [findParentInTable, personToRender.fatherName]);
```